js/tests/unit/dropdown.js

```diff
@@ -342,19 +343,32 @@ $(function () {
 
     var $input = $('#input')
     var $textarea = $('#textarea')
+    var $contenteditable = $('#contenteditable')
+
+    var keys = [38,40,27,32]
 
     $dropdown
       .parent('.dropdown')
       .on('shown.bs.dropdown', function () {
         assert.ok(true, 'shown was fired')
 
-        $input.trigger('focus').trigger($.Event('keydown', { which: 38 }))
-        assert.ok($(document.activeElement).is($input), 'input still focused')
+        keys.forEach(function (key) {
```

Comment:
Don't use `Array.forEach`. It's not supported in all the browsers that Bootstrap 3 supports.